services/ffc/ffc-demo/ffc-demo-claim-service/pre-deploy/post-migration/prd/01/post-migration-script-patch.yaml

```diff
@@ -29,4 +29,4 @@ spec:
         - name: SUBSCRIPTION_NAME
           value: ${SUBSCRIPTION_NAME}
         - name: DB_MIGRATION_VERSION
-          value: "0.1.0" # {"$imagepolicy": "flux-config:ffc-demo-claim-service-dbmigration-dev-01:tag"}
+          value: "5.1.27" # {"$imagepolicy": "flux-config:ffc-demo-claim-service-dbmigration-snd-01:tag"}
```

Comment:
@aadev1 image policy pointing to snd-01. probably the corresponding issue is on the automation template

Reply:
@jeevankuduvaravindran Good spot - updated now